linux_os/guide/system/software/gnome/gnome_screen_locking/dconf_gnome_screensaver_lock_enabled/ansible/shared.yml

```diff
@@ -1,8 +1,14 @@
-# platform = multi_platform_rhel,multi_platform_fedora,multi_platform_ol
+# platform = multi_platform_rhel,multi_platform_fedora,multi_platform_ol,multi_platform_sle
 # reboot = false
 # strategy = unknown
 # complexity = low
 # disruption = medium
+
+{{% if product in ["sle12", "sle15"] %}}
+- name: Dconf Update
```

Comment:
Our rule for SLES-12-010060 has:

Fix Text: This command must be run from an X11 session; otherwise, the command will not work correctly.

Configure the SUSE operating system to allow the user to lock the graphical user interface.

Run the following command to configure the SUSE operating system to allow the user to lock the graphical user interface:

# gsettings set org.gnome.desktop.lockdown disable-lock-screen false

In our testing in SLE-12 we found that we needed Dconf Update to be done or the later set would fail.